recipe/migrations/libprotobuf4233_libgrpc156.yaml

```diff
@@ -1,11 +1,11 @@
 __migrator:
   build_number: 1
   kind: version
-  migration_number: 2
+  migration_number: 1
```

Comment:
Shouldn't we increase the number instead of decreasing it?

Reply:
Or is this different as it stops one and starts another migration?

Reply:
I renamed the migration, so the PR names are less confusing. Rather than a 3rd "Rebuild for protobuf423", it would be "Rebuild for libprotobuf4233_libgrpc156".

Reply:
> Or is this different as it stops one and starts another migration?

Exactly